data/defaults.yaml

```diff
@@ -6,6 +6,7 @@ prometheus::pushprox_client::group: 'pushprox_client'
 prometheus::pushprox_client::package_ensure: 'latest'
 prometheus::pushprox_client::package_name: 'pushprox_client'
 prometheus::pushprox_client::user: 'pushprox_client'
+# renovate: depName=prometheus-community/PushProx
```

Comment:
is there a chance that renovate can parse the puppet code? we moved most of the data out of defaults.yaml into the puppet classes so puppet-strings can pick up the data. We just didn't complete this migration yet.

Reply:
Well the whole thing is based on regex. So if we can harmonize all modules to use the same format, eg:
```
 String[1] $version                                         = 'x.x.x',
```
that is should be doable. In my mind it was much nicer to have the versions centralized in the module-data. Do we need the versions in the `REFERENCE.md`?